sktime/datatypes/_panel/_check.py

```diff
@@ -1184,7 +1184,47 @@ def _check_polars_panel(obj, return_metadata=False, var_name="obj", scitype="Pan
 
 
 class PanelGluontsList(ScitypePanel):
-    """Data type: polars.DataFrame based specification of panel of time series.
+    """Data type: gluonTS representation of univariate and multivariate time series.
+
+    Name: ``"gluonts_ListDataset_panel"``
+
+    Short description:
+
+    A ``list`` of ``dict``,
+    with list index = instances, ``dict['target']`` rows = time points,
+    ``dict['target']`` cols = variables, and ``dict['start']`` marking the interval.
+    Similar to ``gluonts.dataset.common.ListDataset``.
+
+    Long description:
+
+    The ``"gluonts_ListDataset_panel"`` :term:`mtype` is a concrete specification
+    that implements the ``Panel`` :term:`scitype`, i.e., the abstract
+    type of a collection of time series.
+
+    An object ``obj: list`` follows the specification iff:
+
+    * structure convention: ``obj`` must be a list of ``dict``.
+      Each ``dict`` contains two keys, ``"start"`` and ``"target"``. 
+      ``"start"`` maps to the interval (e.g., `pandas.Period`) of the series. 
```

Comment:
can we be clearer what exactly must be the values under the `"start"` and `"target"` dict keys?

Reply:
Yes. Changed this to be more specific. It also appears that ``"start"`` key is optional per the code, so changed that too.